src/ci/docker/x86_64-gnu-nopt/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   cmake \
   sudo \
   gdb \
+  p7zip-full \
```

Comment:
Uhm... Actually I might have removed one too many: does the `cross` image also produce tarballs? Even if it is not `dist-*`, it is flagged `DEPLOY=1`.